mlir/lib/Dialect/Rock/Tuning/GridwiseGemmParams.cpp

```diff
@@ -493,6 +496,15 @@ const InitParams &PopulateParamsXDL::getUniversalParameters() const {
   return universalParameters;
 }
 
+int64_t PopulateParamsXDL::getUniversalKPack() const {
+  // TODO(grossini): this is because we are using universal parameters
```

Comment:
Hold on, can you make sure that the heuristic parameters list doesn't have the invalid config?

Reply:
Unfortunately, we cannot remove the invalid config from the list, because it is valid for `fp32`. It is only for `fp16` that `32, 64, 4, 32, 64, 1` becomes invalid